readme.md

```diff
@@ -19,6 +19,20 @@ npm install flightradar24-client
 
 ### `radar(north, west, south, east)`
 
+Here they represent a geographical bounding box (in decimal degrees) with: 
+
+```
+ Northern edge latitude
+ Western edge longitude
+ Southern edge latitude
+ Easter edge longitude
+```
+
+If you want to test it from command line, you can do it by adding the shebang `#!/usr/bin/node` 
```

Comment:
I would like to keep this out of the readme. IMO it is too general. Also, there are many non-command-line use cases.